gtest_parallel_tests.py

```diff
@@ -376,16 +376,18 @@ def test_running_parallel_tests(self):
 
 class TestTestTimes(unittest.TestCase):
   def test_race_in_test_times_load_save(self):
-    max_number_of_workers = 16
-    max_numver_of_read_write_cycles = 128
+    max_number_of_workers = 1
```

Comment:
Sorry, I changed this for debugging and it looks like my testing was not complete. :(